### ISSUE TITLE
Inaccurate Indentation Failures

### ISSUE THREAD

**Issue by allenhartwig** [sasstools/sass-lint#611]
Lint fails on indentation with odd results. Eg: "Indentation of 0..." (when the line does have indentation) "..., expected 2" (what it has)

**What version of Sass-Lint are you using?**
1.5.1

**Please show your full configuration:**
sass-lint.yaml
```yml
rules:
  indentation:
    - 2
```

**What did you do? Please include the actual source code causing the issue.**
`sass-lint -c sass-lint.yaml test.sass -v -q -f compact`
test.sass
```css
table
  width: 100%

  a
    &:link
      text-decoration: none

    &:visited
      text-decoration: none

    &:active
      text-decoration: underline
```
**What did you expect to happen?**
Lint passes

**What actually happened? Please include the actual, raw output from ESLint.**
Lint fails with 
```bash
test.sass: line 4, col 1, Error - Indentation of 0, expected 2 (indentation)
test.sass: line 5, col 1, Error - Indentation of 0, expected 2 (indentation)
test.sass: line 6, col 1, Error - Indentation of 0, expected 4 (indentation)
test.sass: line 9, col 1, Error - Indentation of 0, expected 4 (indentation)
test.sass: line 12, col 1, Error - Indentation of 0, expected 4 (indentation)
```

**Comment by DanPurdy** [comment]
There's quite a few issues surrounding this already yes. Sass syntax is notorious for it as well as using crlf line endings or hard tabs.. The other issues have a more in depth description but unfortunately until we can get 1.6 out and investigate this with the latest version of the AST we use we can't fix it, out hands are tied.. I recommend just disabling it for now unfortunately if you match any of the criteria I mention above. :sob:

**Comment by allenhartwig** [comment]
I'm using LF and soft tabs (2 spaces). Should I still be seeing this issue?

**Comment by DanPurdy** [comment]
You're using sass syntax not scss right? There's still issues with scss in certain places but it's definitely more stable than sass syntax purely down to the differences the AST we're using handles those formats.

**Comment by allenhartwig** [comment]
Yes, sass syntax.

Are there any other known issues right now, or is it just indentation?

**Comment by Marc-Boisvert-Dupras** [comment]
I'd just like to add I'm seeing this as well (I guess that's to be expected at the moment) using Sass syntax.

**Comment by Kathrynwatts** [comment]
Wanted to add I am seeing this, with scss syntax. If I do 2 spaces it will fix the issue, but one tab will trigger a warning. I'm only having this issue in my partials though (files starting with _) my main.scss file is not triggering this warning at all. I'm using sass-lint 1.4.2 in Atom.

**Comment by DanPurdy** [comment]
@Kathrynwatts Tabs were never supported in the indentation rule as described in the documentation, this was purely down to the fact that until our 1.6.0 update the AST we use didn't support tabs.

I've just updated this rule to hopefully fix the majority of issues we have with indentation and also to allow you to specify the tab character. Should be available with our 1.8.0 update

**Comment by DanPurdy** [comment]
@allenhartwig this will be fixed in our 1.8 update. Closing as it's already merged in ready to go.
